.azure/hpu-tests.yml

```diff
@@ -79,10 +79,18 @@ jobs:
       condition: eq(variables['pkg_source'], 'source')
       displayName: 'OverInstall lightning from source'
 
-    - bash: pip uninstall -y pytorch-lightning
+    - bash: |
+        pip uninstall -y pytorch-lightning
+        pip list
       condition: eq(variables['dependency'], 'lightning')
       displayName: 'drop PL package'
 
+    - bash: |
+        pip uninstall -y lightning
+        pip list
+      condition: eq(variables['dependency'], 'pytorch-lightning')
+      displayName: 'drop Lightning package'
+
     - bash: |
         # todo: consider test all files not listed as you may easily forget to add new
         python -m pytest -sv \
```

Comment:
It seems that test_hpu_precision.py is not included in the list of files here. Not sure if there are others missing

Reply:
Yep, fabric is not yet fully supported.